nvtabular/loader/tensorflow.py

```diff
@@ -188,6 +189,12 @@ class KerasSequenceLoader(tf.keras.utils.Sequence, DataLoader):
     - reader_kwargs: dict
         extra kwargs to pass when instantiating the underlying
         `nvtabular.Dataset`
+    sparse_list : list(str) or None
+        list with column names of columns that should be represented as sparse tensors
+    sparse_max : dict
+        dictionary of key: column_name + value: integer representing max sequence length for column
+    sparse_dense : bool
+        bool value to activate transforming sparse tensors to dense
```

Comment:
I think this API is a little confusing - like we are specifying which columns to treat as sparse, and then have a flag to convert to dense.

Can we simplify this at all? Like if we're producing dense outputs,  would it be possible to just add the columns to the continuous columns?

Reply:
resolved over slack - going to leave as is with some minor name parameter name changes